api/src/pcapi/core/users/models.py

```diff
@@ -751,3 +754,18 @@ class UserSuspension(PcObject, Base, Model):
 class UserSession(PcObject, Base, Model):
     userId: int = sa.Column(sa.BigInteger, nullable=False)
     uuid: UUID = sa.Column(postgresql.UUID(as_uuid=True), unique=True, nullable=False)
+
+
+class UserProFlags(PcObject, Base, Model):
+    __tablename__ = "user_pro_flags"
+
+    firebase: dict = sa.Column(
+        MutableDict.as_mutable(postgresql.json.JSONB), nullable=True, default={}, server_default="{}"
```

Comment:
Pourrait-on ajouter une contrainte `NOT NULL` sur la colonne ? Ça éviterait le risque de fournir explicitement (en Python) une value `None` (par erreur, probablement).